protostar/protostar_toml/protostar_toml_section.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+from typing import Dict, List, Union
+
+
+class ProtostarTOMLSection(ABC):
+    TOMLCompatibleDict = Dict[str, Union[str, int, bool, List[str]]]
```

Comment:
`Compatible` is a bit weird as it suggests that other dicts aren't compatible. Compatibilty relates more to interfaces not typing. `ParsedProtostarTOML` idk?